ext/standard/tests/file/bug71882.phpt

```diff
@@ -3,8 +3,11 @@ Bug #71882 (Negative ftruncate() on php://memory exhausts memory)
 --FILE--
 <?php
 $fd = fopen("php://memory", "w+");
-var_dump(ftruncate($fd, -1));
+try {
+    var_dump(ftruncate($fd, -1));
```

Comment:
All the `var_dump()`s in try-catch blocks *could* be removed as it's sure that an exception is thrown before executing them. At least this is how I prefer to do it, so feel free to ignore. Otherwise, the PR looks good for me :)

Reply:
The reason I don't like removing them is *if* we remove the Error/Exception you need to go back up and add it to make sure that the behaviour is expected.

Also everything is just "standard". Moreover, this can be removed after the fact.